README.md

```diff
@@ -178,23 +178,23 @@ By default, redbiom is setup to query against [Qiita](https://qiita.ucsd.edu). F
 
 Now that we have some samples, let's pull out their sample data. Qiita contains a huge amount of data, which are logically partitioned by the sample preparations and processing parameters -- these partitions are denoted as **contexts** in redbiom. In order to pull out the data, we need to specify the context to operate in. There are a lot of contexts, so let's filter to only those which are 16S and V4 using `grep`. We're also going to `cut` the first three columns of data as the fourth one is a voluminous description of the processing parameters. And last, let's `sort` the results by the number of samples represented in the context. Unfortunately, the `grep` removes the column headers, so we'll run a second summarize command and just grab the header:
 
-    $ redbiom summarize contexts | cut -f 1,2,3 | grep 16S-v4 | sort -k 2 -n
-    Pick_closed-reference_OTUs-illumina-16S-v45-66f541  102 29598
-    Pick_closed-reference_OTUs-flx-16S-v4-66f541    116 4699
-    Pick_closed-reference_OTUs-ls454-16S-v4-66f541  145 8437
-    Pick_closed-reference_OTUs-titanium-16S-v46-66f541  214 3568
-    Pick_closed-reference_OTUs-titanium-16S-v4-66f541   800 14269
-    deblur-workflow-illumina-16S-v4-150nt-ae489c    24613   1932042
-    deblur-workflow-illumina-16S-v4-100nt-ae489c    60150   3738847
-    deblur-workflow-illumina-16S-v4-90nt-ae489c 65143   3162632
-    Pick_closed-reference_OTUs-illumina-16S-v4-66f541   89405   84828
-    
+    $ redbiom summarize contexts | cut -f 1,2,3 | grep 16S-v4 | grep Greengenes-illumina |  sort -k 2 -n
+    Pick_closed-reference_OTUs-Greengenes-illumina-16S-v45-100nt-a243a1	22	8178
+	Pick_closed-reference_OTUs-Greengenes-illumina-16S-v45-90nt-44feac	22	8471
```

Comment:
Would it be possible to use spaces instead of tabs here and in the indented blocks in the "Retrieving pre-selected samples" section?

Reply:
Of course! Thank you